db/mysql/ops.go

```diff
@@ -49,19 +50,24 @@ func (c *client) SendMessage(ctx context.Context, id string, message *domain.Mes
 	return nil
 }
 
-func (c *client) ListChatRoom(_ context.Context) ([]*domain.ChatRoom, error) {
```

Comment:
In the function GetChatroom, I don't understand why we have joins. We have a relationship between users and Chatroom, where chatroom.user is an array of strings. I assume we have User IDs stored in that array. Please explain this.

Reply:
Getting chatroom object from the database is enough  because there is already users list in that

Reply:
You are right. Removed the joins.